js/selectors/index.js

```diff
@@ -84,3 +84,33 @@ export const getVisibleVolcanoes = createSelector(
     return result
   }
 )
+
+// Eruptions
+
+export const getVisibleEruptions = createSelector(
+  [eruptionsEnabled, eruptionsData, filters, crossSectionPoints],
+  (eruptionsEnabled, eruptionsData, filters, crossSectionPoints) => {
+    if (!eruptionsEnabled) {
+      return []
+    }
+    const minTime = filters.get('minTime')
+    const maxTime = filters.get('maxTime')
+    // const crossSectionFilter = getCrossSectionFilter(crossSectionPoints)
```

Comment:
I think that's the reason why you see all the eruptions in the cross-section.

Reply:
Thanks! I had commented that out during testing and never uncommented. Well spotted!